x-pack/plugins/case/server/client/cases/push.ts

```diff
@@ -84,7 +78,7 @@ export const push = async ({
     ]);
   } catch (e) {
     const message = `Error getting case and/or connector and/or user actions: ${e.message}`;
-    throw createError(e, message);
+    throw createCaseError({ message, error: e, logger });
```

Comment:
Using a `CaseError` for these so that we can boomify it once we get to the route handler and keeping things consistent.